src/domain/notifications/notifications.repository.v2.ts

```diff
@@ -104,23 +129,29 @@ export class NotificationsRepositoryV2 implements INotificationsRepositoryV2 {
   }): Promise<
     Array<{
       subscriber: `0x${string}`;
-      deviceUuid: Uuid;
+      deviceUuid: UUID;
       cloudMessagingToken: string;
     }>
   > {
-    return this.notificationsDatasource.getSubscribersBySafe(args);
+    return this.notificationsDatasource.getSubscribersBySafe({
+      chainId: args.chainId,
+      safeAddress: args.safeAddress,
+    });
   }
 
   deleteSubscription(args: {
-    deviceUuid: Uuid;
+    deviceUuid: UUID;
     chainId: string;
     safeAddress: `0x${string}`;
-    signerAddress: `0x${string}`;
   }): Promise<void> {
-    return this.notificationsDatasource.deleteSubscription(args);
+    return this.notificationsDatasource.deleteSubscription({
+      deviceUuid: args.deviceUuid,
+      chainId: args.chainId,
+      safeAddress: args.safeAddress,
+    });
   }
```

Comment:
Nit: we could just still `return this.notificationsDatasource.deleteSubscription(args)` here, right?

Reply:
Changed in 2a1216101fd50bafd404c24e2a6fddae907dcbf4.